application/django/README.md

```diff
@@ -0,0 +1,24 @@
+Init the SecretHub demo repo with example values
```

Comment:
I think we can extend a little on the overall structure of this readme. For example:
```
# Django application in Docker
<a description of what we're going to run here>

## Prerequisites
1. Docker installed and running

## Running the example
<the steps already in this document>
```

Reply:
I agree on this. I will add this